pkg/cli/cli.go

```diff
@@ -193,6 +212,59 @@ func WithCompletion(c *cli) error {
 	return nil
 }
 
+//set default descriptions for root command
+func (c *cli) setRootCommandDefaults() error {
+	if c.short == "" {
+		c.short = "Development kit for building Kubernetes extensions and tools."
+	}
+	if c.long == "" {
+		c.long = fmt.Sprintf(`Development kit for building Kubernetes extensions and tools.
+	
+		Provides libraries and tools to create new projects, APIs and controllers.
+		Includes tools for packaging artifacts into an installer container.
+		
+		Typical project lifecycle:
+		
+		- initialize a project:
+		
+		%s init --domain example.com --license apache2 --owner "The Kubernetes authors"
+		
+		- create one or more a new resource APIs and add your code to them:
+		
+		%s create api --group <group> --version <version> --kind <Kind>
+		
+		Create resource will prompt the user for if it should scaffold the Resource and / or Controller. To only
+		scaffold a Controller for an existing Resource, select "n" for Resource. To only define
+		the schema for a Resource without writing a Controller, select "n" for Controller.
+		
+		After the scaffold is written, api will run make on the project.
+		`, c.commandName, c.commandName)
```

Comment:
Don't add tabs to multi-line strings, and `%[1]s` form is clearer than having to repeat the argument to `fmt.Sprintf`:
```suggestion
		c.long = fmt.Sprintf(`Development kit for building Kubernetes extensions and tools.

Provides libraries and tools to create new projects, APIs and controllers.
Includes tools for packaging artifacts into an installer container.

Typical project lifecycle:

- Initialize a project:

%[1]s init --domain example.com --license apache2 --owner "The Kubernetes authors"

- Create one or more a new resource APIs and add your code to them:

%[1]s create api --group <group> --version <version> --kind <Kind>

Create resource will prompt the user for if it should scaffold the Resource and / or Controller.
To only scaffold a Controller for an existing Resource, select "n" for Resource. To only define
the schema for a Resource without writing a Controller, select "n" for Controller.

After the scaffold is written, api will run make on the project.
`,
			c.commandName)
```